diff -uNr ipw3945-1.1.0-pre2.org/Makefile ipw3945-1.1.0-pre2/Makefile
--- ipw3945-1.1.0-pre2.org/Makefile	2006-06-24 01:35:00.000000000 +0200
+++ ipw3945-1.1.0-pre2/Makefile	2006-08-01 03:28:54.000000000 +0200
@@ -70,8 +70,8 @@
 		$(IEEE80211_INC))
 endif
 
-IEEE80211_BASE := $(shell var=($(IEEE80211_RES)) ; echo $${var[1]})
-IEEE80211_PATH := $(shell var=($(IEEE80211_RES)) ; echo $${var[0]})
+IEEE80211_BASE := $(shell echo "$(IEEE80211_RES)"|awk -F" " '{print $2}')
+IEEE80211_PATH := $(shell echo "$(IEEE80211_RES)"|awk -F" " '{print $1}')
 
 # If the ieee80211 subsystem is not found in the default kernel
 # build location then we need to add the include path and 
