diff -uNr ipw3945-1.1.0.org/Makefile ipw3945-1.1.0/Makefile
--- ipw3945-1.1.0.org/Makefile	2006-07-14 15:34:55.000000000 +0000
+++ ipw3945-1.1.0/Makefile	2006-08-06 18:43:31.000000000 +0000
@@ -237,7 +237,7 @@
 	rm -f tags TAGS
 
 modules:
-	$(MAKE) -C $(KSRC) M=$(DIR) $(IEEE80211_EXTRA) modules
+	$(MAKE) -C $(KSRC) M=$(DIR) $(IEEE80211_EXTRA) MODVERDIR=$(IEEE80211_MODVERDIR) modules
 
 add_radiotap:
 	@snapshot/add_radiotap $(KSRC)
